pandas/core/generic.py

```diff
@@ -1144,6 +1139,23 @@ def rename_axis(self, mapper, axis=0, copy=True, inplace=False):
         deprecated and will be removed in a future version. Use ``rename``
         instead.
 
+        ``DataFrame.rename_axis`` supports two calling conventions
+
+        * ``(index=index_mapper, columns=columns_mapper, ...)``
+        * ``(mapper, axis={'index', 'columns'}, ...)``
+
+        The first calling convention will only modify the names of
+        the index and/or the names of the index backing the columns.
```

Comment:
same: index -> Index, or just remove.

Reply:
changed to "names of the Index object that is the columns."